os/mm/mm_heap/mm_memalign.c

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2016 Samsung Electronics All Rights Reserved.
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
It is the first year that this file is generated. So we don't need to modify this.

Reply:
ok